### ISSUE TITLE
Detect first email access screen

### ISSUE THREAD

**Issue by Odrakir** [toggl/mobileapp#7010]
When the user picks email signup/login from the main screen we have to decide if we want to forward him first to the signup screen or to the login screen.

By default it'll go to signup, but if we detect the user has previously signed up with this app we'll send him to login.

Find a way to know that, we probably are saving something in the local storage that can give us that information, if not, we'll have to add some attribute for this.

Add this logic to the onboarding ViewModel (the first screen of the app)

#7003 and #7006 should be done first.

Designs and full specs: https://www.notion.so/toggl/Designs-wip-d767ebc157c542c790e1bc93eeaf006f#835299ea40344044b2572f356b4f47c1

**NOTE:** This issue is part of the `Onboarding refactor` project and it should be merged into the project branch and not into develop.

**Comment by daividssilverio** [comment]
As discussed here https://toggl.slack.com/archives/CTJJ98Z4J/p1581612509031400
We could use the page where the user has left onboarding (tapped on one of the `continue` options) to assume if the user is new or not.

For example, if the user taps `continue with e-mail` on the first page, we could assume the user already has an account, otherwise we can redirect them to the sign up screen.

**Comment by dshr** [comment]
I think might've been handled in #7115 by using the `lastTimeUsageStorage.LastLogin` functionality. Shall we close this, @Odrakir?

**Comment by sofokli1** [comment]
As Des said, this looks like it is already done in the onboarding VM https://github.com/toggl/mobileapp/pull/7115/files#diff-af00ba5710fc34fbc7228d0644f8a1dcR93
